app/controllers/well_known/host_meta_controller.rb

```diff
@@ -7,7 +7,14 @@ class HostMetaController < ActionController::Base # rubocop:disable Rails/Applic
     def show
       @webfinger_template = "#{webfinger_url}?resource={uri}"
       expires_in 3.days, public: true
-      render content_type: 'application/xrd+xml', formats: [:xml]
+
+      respond_to do |format|
+        format.any do
+          render content_type: 'application/xrd+xml', formats: [:xml]
+        end
+
+        format.json
```

Comment:
I see why you did it this way, but this whole thing is so small it seems weird to give it a builder style view ... wonder about just inlining the hash here and calling to_json in this response?

It's like, just barely big enough that I could see either way.

It'd be DELIGHTFUL if there were some data structure that we could call to_xml and to_json on to get the correct output and delete both view templates.

Reply:
Ok, let's inline it :-)

Unfortunately, JRD is designed specifically for XRD with some quirky rules, so we cannot use a general-purpose converter from XML to JSON.